input/fsh/instances/QuestionnaireResponse-patients-fetal-death-carmen-lee.fsh

```diff
@@ -68,18 +68,12 @@ Usage: #example
   * answer.valueCoding = $HL7_EducationLevelCS#SEC "Some secondary or high school education"
 * item[+]
   * linkId = "mother-ethnicity"
-  * text = "Are you Spanish/Hispanic/Latina? If not Spanish/Hispanic/Latina, check the “No” box. If Spanish/Hispanic/Latina, check the appropriate box."
-  * item
-    * linkId = "mother-shl"
-    * text = "Are you Spanish/Hispanic/Latina"
-    * answer.valueCoding = urn:oid:2.16.840.1.113883.6.238#2135-2 "Hispanic or Latino"
+  * answer.valueBoolean = true
```

Comment:
The questionnaire says: "9. Are you Spanish/Hispanic/Latina? (if so, please answer 9a-9d)"

I imagine the ethnicity type should be addressed in a-d in the example.

Reply:
Good point....Adding detailed ethnicity to the example questionnaire to line up with the example in FHIR.

Not sure what you mean by "text should be removed"?